airbyte-cdk/java/airbyte-cdk/db-sources/src/main/kotlin/io/airbyte/cdk/integrations/source/relationaldb/InitialLoadTimeoutUtil.kt

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.cdk.integrations.source.relationaldb
+
+import com.fasterxml.jackson.databind.JsonNode
+import io.github.oshai.kotlinlogging.KotlinLogging
+import java.time.Duration
+import java.util.*
+
+private val LOGGER = KotlinLogging.logger {}
+
+object InitialLoadTimeoutUtil {
+
+    val MIN_INITIAL_LOAD_TIMEOUT: Duration = Duration.ofHours(4)
+    val MAX_INITIAL_LOAD_TIMEOUT: Duration = Duration.ofHours(24)
+    val DEFAULT_INITIAL_LOAD_TIMEOUT: Duration = Duration.ofHours(8)
+
+    @JvmStatic
+    fun getInitialLoadTimeout(config: JsonNode): Duration {
+        val isTest = config.has("is_test") && config["is_test"].asBoolean()
```

Comment:
nit: is this just to prevent logging in tests?  Is there not a way to tell the test to skip WARN-level messages, or maybe these should be INFO?

Reply:
Actually this in case we want to set the time outside of these bounds in a test environment - the checks for range only apply for a prod environment